Packs/Genians/pack_metadata.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "Genians",
+    "description": "Use the Genians integration to block IP using assign tag and unassign tag.",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
```

Change to Genians support
```suggestion
    "email": "support@genians.com",
```